be/src/exec/parquet_writer.cpp

```diff
@@ -88,6 +88,16 @@ Status RollingAsyncParquetWriter::append_chunk(Chunk* chunk, RuntimeState* state
     return _writer->write(chunk);
 }
 
+Status RollingAsyncParquetWriter::cancel(RuntimeState* state) {
+    if (_writer != nullptr) {
+        auto st = _fs->delete_file(_writer->file_location());
```

Comment:
async io tasks may still be in execution, or just queueing.

Reply:
another issue is that `RollingAsyncParquetWriter` may write multiple files. 

Do you think we can also clean the previously written files?

Reply:
done